code/game/gamemodes/events/biomass.dm

```diff
@@ -202,10 +202,11 @@
 		var/area/Hallway = locate(type)
 
 		for(var/turf/simulated/floor/Floor in Hallway.contents)
-			if(Floor.contents.len <= 0)
+			if(Floor.contents.len <= 1)//There's always an /atom/movable/lighting_overlay on turfs

```

Comment:
Vine spreading doesn't have to do with density does it? The way it's always worked until now is that it picked a turf with "nothing" on it, but since now all turfs have a lighting overlay, you gotta add +1.